doc/source/robot/example.robot

```diff
@@ -16,6 +16,13 @@ First Test
 Undocumented Test
                         Should Be Equal     ${MESSAGE}    Hello, world!
 
+Test with raw docs
+    [Documentation]     *RAW* An example docstring of which the line endings get preserved.
```

Comment:
Any reasons why we don't just preserve line endings by default and let rst parsers deal with it?

Reply:
I did it because it produces better looking reStructuredText. Do you want the tool to always preserve line endings?

Reply:
We decided to always preserve line endings. Indentation, however, is not preserved.

Reply:
Since Robot Framework 3.2 the indents can be preserved, so that's what we'll require and use.